Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -515,6 +527,9 @@ def get_email_reputation(email, threshold=None, status="active,inactive"):
     """
     params = build_params(value=email, type="email", status=status, limit=0)
     indicator = search_indicator_by_params(params, email)
+    if not indicator:
+        return
```

Comment:
in this case you are not returning anything (no entry), this will cause command timeout.

relevant for most of your changes.

Reply:
@yaakovi  I am returning entry in inner function